Adyen/UI/List/ListItem.swift

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2021 Adyen N.V.
+// Copyright (c) 2018 Adyen N.V.
```

Comment:
Traveling back in time? :P

Reply:
v4 `swiftformat` apparently uses file creation date for this job